src/components/network/network.js

```diff
@@ -152,6 +152,10 @@ genotet.NetworkView = function(viewName, params) {
     .on('genotet.nodeClick', function(event, node) {
       this.panel.displayNodeInfo(node);
       this.loader.incidentEdges(node);
+      this.signal('link', {
```

Comment:
Besides clicking a node and an edge don't seem to be a same type of event at all. Your grouping would make unnecessary implicit nesting for event handling.